djangoProject/purchase/admin.py

```diff
@@ -1,3 +1,10 @@
 from django.contrib import admin
 
-# Register your models here.
+from purchase.models import Purchase
+
+
+@admin.register(Purchase)
+class PurchaseAdmin(admin.ModelAdmin):
+    list_display = [Purchase.__str__, 'id', 'user_id', 'book_id', 'date']
```

Comment:
the task was not correct. It's ok if `list_display` is used instead of `__str__`